lib/rubocop/cop/rspec/no_expectation_example.rb

```diff
@@ -28,7 +28,34 @@ module RSpec
       #     expect(a?).to be(true)
       #   end
       #
+      # This cop can be customized allowed expectation methods pattern
+      # with `AllowedPatterns`. \Aexpect_ and \Aassert_ are allowed by default.
+      #
+      # @example `AllowedPatterns` configuration
+      #
+      #   # .rubocop.yml
+      #   # RSpec/NoExpectationExample:
+      #   #   AllowedPatterns:
+      #   #     - \Aexpect_
+      #   #     - \Aassert_
```

Comment:
By the way, should it be double escaped? I see [just `\A` as a suggestion in RuboCop's own 'config/default.yml` here](https://github.com/rubocop/rubocop/blob/d1270c468cdb9a211229e36aef8d568ae3bfe607/config/default.yml#L2786). I couldn't find any other examples there.

I don't have a strong preference, apart from double escaping.
But just to make sure the default config's double escaping is correct, I'd test this change live, just by running `rubocop` on a code that has `expect_`/`allow_` to see if the cop ignores those, as we stub the config in our spec.

Reply:
Indeed it is. I will change it to ``^``.

> By the way, should it be double escaped? I see [just \A as a suggestion in RuboCop's own 'config/default.yml` here](https://github.com/rubocop/rubocop/blob/d1270c468cdb9a211229e36aef8d568ae3bfe607/config/default.yml#L2786). I couldn't find any other examples there.

Oh, I didn't escape there, but I think it may have been escaped by ConfigFormatter.